Update Readme (#1)
DevExpress-Examples/winforms-pivotgrid-linq-to-sql-server-mode: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T828611)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# PivotGridControl and LINQ to SQL - a Server Mode Example

> This example demonstrates the PivotGridControl that is bound to LINQ to SQL classes and operates in Server mode.

Reviewer: ```suggestion
This example demonstrates a PivotGridControl bound to LINQ and SQL classes and operates in server mode.
```
